Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Mesh/AtomMeshBus.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/EntityId.h>
+#include <Atom/Feature/Mesh/MeshFeatureProcessorInterface.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        //! Bus for retrieving data about a given entity's Atom mesh.
+        class AtomMeshRequests
+        : public AZ::EBusTraits
+        {
+        public:
+            static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+            static const EBusAddressPolicy AddressPolicy = EBusAddressPolicy::ById;
+            using BusIdType = EntityId;
+
+            //! Returns the handle to the Atom mesh.
+            virtual const MeshFeatureProcessorInterface::MeshHandle* GetMeshHandle() const = 0;
```

Comment:
question: I don't know too much about Atom but want to check should the `MeshHandle` itself actually be a pointer? Or could this be returned by value (or is it an alias for an actual Mesh itself)